objects/screen.c

```diff
@@ -21,6 +21,22 @@
 
 /** awesome screen API
  *
+ * Screen objects can be added and removed over time. To get a reliable callback,
+ * use `awful.screen.connect_for_each_screen`:
```

Comment:
I would word this differently. Something like "To get a callback for all current and future screens", or something like that. If the user just wants the future screens, they can use `screen.connect_signal("added", f)`.